general/general_shacl.ttl

```diff
@@ -0,0 +1,101 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
```

Comment:
see comments from above. 

It seems that there are both general and General folder right now:
<img width="638" alt="image" src="https://github.com/GAIA-X4PLC-AAD/ontology-management-base/assets/70939848/3c8216db-90db-4a23-8e2d-335e6e31f44f">

Reply:
seems to be fixed

Reply:
yes, General is gone now.